### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-270.16][WIP] Logo: UI is triggered by the down-event.

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11294]
#### Is this a bug, enhancement, or feature request?
Bug
#### Describe your proposal.
On accessing the image link, the UI gets activated without releasing the enter key.
every UI element should be activated only when the key is released. If the key is on hold, the UI should not be activated. 

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Use any image, UI will gets activated without releasing the enter key

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

**Comment by JyotiKumari1507** [comment]
@droshev We can discuss on this more in next call

**Comment by droshev** [comment]
@JyotiKumari1507 we need the input written so it will be easier to process

**Comment by JyotiKumari1507** [comment]
@droshev  refer screenshot shared by accessibility team.  As per their testing, On accessing the ‘SAP' logo link, the UI gets activated without releasing the enter key. UI should only activate with releasing the enter key.
<img width="903" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/157268666/f6fe7c5d-f187-421e-b7d0-776feac9772d">

**Comment by droshev** [comment]
Most of the fields are not filled.

**Comment by JyotiKumari1507** [comment]
@droshev please let us know what fields are pending to fill from our side

**Comment by droshev** [comment]
@JyotiKumari1507 did you set all aria attributes needed? If so, can you create a small stackblitz example demonstrating that you cannot set this on the application side?
I tested the same on our doc page and the mouse down nothing happens. The navigation gets triggered once i release the mouse click

**Comment by divyamalhotra488** [comment]
Hi @droshev, the issue is seen with keydown event. Mouse down is working fine for us as well.

**Comment by droshev** [comment]
@divyamalhotra488 Check this [stackblitz](https://stackblitz.com/edit/stackblitz-starters-ouzarw?file=package.json) - there is a simple link (html) and if you focus and simulate keydown it will act the same. It won't wait for keyup

**Comment by divyamalhotra488** [comment]
Hi @droshev, so is this expected behavior since it is not falling within accessibility standards ?

**Comment by MeghanaVernekar** [comment]
Hello Colleagues,
There is an accessibility requirement (270.16) that No UI should get triggered with only down event and should not be activated until the user releases the key. For more info about the requirement attached the link below: https://wiki.one.int.sap/wiki/display/Acc/ACC-270+Details#ACC270Details-acc27016.

But I have tried retesting the issue now with the updated URL: https://qa.cobalt.ariba.com/emc/ and observed that the **SAP logo** button is not reachable with keyboard which is again an issue.

Regards,
Meghana.

**Comment by droshev** [comment]
Can you open a new ticket and creating a stackblitz with the issue?

**Comment by divyamalhotra488** [comment]
Created new defect: https://github.com/SAP/fundamental-ngx/issues/12044
